heartbeat/Filesystem

```diff
@@ -389,6 +389,82 @@ fstype_supported()
 	return $OCF_ERR_INSTALLED
 }
 
+#
+# Determine backend filesystem support state
+#
+backend_support()
+{
+	case "$FSYPE" in
+		overlay)
+			# it should fail immediately if kernel module not found
+			modinfo overlay >/dev/null 2>&1
+			;;
+		overlay)
```

Comment:
Unless I'm mistaken, the second "overlay" case would never match. Is this supposed to be "overlayfs"?